dcpy/utils/postgres.py

```diff
@@ -8,9 +8,15 @@
 from typing import Optional
 
 
+def _format_schema_name(schema: Optional[str] = None):
```

Comment:
nit: probably wouldn't make this optional. I feel like it should just raise an exception if schema is "" or None.

Reply:
agreed. but since the majority of product builds still use the default `public` schema in their build DB, I didn't wanna force a change in all workflows to declare the schema yet, or distribute this check/formatting to multiple places

the function may not even be worth it since `git.run_name` should always cover the issue of dashes from branch names, so I dropped it in [8da4ecd](https://github.com/NYCPlanning/data-engineering/pull/234/commits/8da4ecd78ff79327fc87b77bfb1d3ec0746323ff)